From 419baceb0c2978fde1b67cbe472899fd1d21612e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Alex=20R=C3=B8nne=20Petersen?= <alex@alexrp.com>
Date: Thu, 4 Sep 2025 00:56:08 +0200
Subject: [PATCH 088/133] std.elf: document which OSABI values are not assigned
 in gABI
X-Developer-Signature: v=1; a=openpgp-sha256; l=1054; i=xtex@astrafall.org;
 h=from:subject; bh=hQEE5wF+SALRbAQuV8t2Aj8zsXTyVKx6eKHpeSuahD0=;
 b=owGbwMvMwCW2U4Ij7wZL9ETG02pJDJn78jhm610+5VST6mCgryM1T+RYwa15/Eum7DmxQTNUL
 i2rtvN0x0QWBjEuBksxRZYiwwZvVp10ftFl5bIwc1iZQIZIizQwMDAwsDDw5SbmlRrpGOmZahvq
 GRrqGOkYMXBxCsBUG/ozMtzKvSrGfclSrUTb/IyR8qFH5o4GmZn8/8uEtBK2cnxJWMXwTz/xz9M
 Lx9ul3bTZRWbwHP/s/OGmtOD/D4t7133+OEFejhcA
X-Developer-Key: i=xtex@astrafall.org; a=openpgp;
 fpr=7231804B052C670F15A6771DB918086ED8045B91

(cherry picked from commit 1c090d3d67bbc0182652eab003aef86438da0d7d)
---
 lib/std/elf.zig | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/lib/std/elf.zig b/lib/std/elf.zig
index e6c4a10ea606..bd9a6b52ac75 100644
--- a/lib/std/elf.zig
+++ b/lib/std/elf.zig
@@ -1122,17 +1122,17 @@ pub const OSABI = enum(u8) {
     CLOUDABI = 17,
     /// Stratus Technologies OpenVOS
     OPENVOS = 18,
-    /// NVIDIA CUDA architecture
+    /// NVIDIA CUDA architecture (not gABI assigned)
     CUDA = 51,
-    /// AMD HSA Runtime
+    /// AMD HSA Runtime (not gABI assigned)
     AMDGPU_HSA = 64,
-    /// AMD PAL Runtime
+    /// AMD PAL Runtime (not gABI assigned)
     AMDGPU_PAL = 65,
-    /// AMD Mesa3D Runtime
+    /// AMD Mesa3D Runtime (not gABI assigned)
     AMDGPU_MESA3D = 66,
-    /// ARM
+    /// ARM (not gABI assigned)
     ARM = 97,
-    /// Standalone (embedded) application
+    /// Standalone (embedded) application (not gABI assigned)
     STANDALONE = 255,
 
     _,
-- 
2.52.0

